sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -689,11 +673,471 @@ export enum KnownAnalyzerNames {
     ZhHantMicrosoft = "zh-Hant.microsoft"
 }
 
+// @public
+export const enum KnownApiVersion20200630 {
+    TwoThousandTwenty0630 = "2020-06-30"
+}
+
+// @public
+export const enum KnownApiVersion20200630Data {
```

Comment:
especially why are there two of them?

Reply:
One is for Data and Other is for Service. We are know exporting either of it.